flow/layers/opacity_layer.cc

```diff
@@ -101,4 +103,8 @@ void OpacityLayer::Paint(PaintContext& context) const {
   PaintChildren(context);
```

Comment:
Painting the children is a waste if alpha is 0 and the saveLayer just above it is a waste if alpha is 255. Skia will likely notice these conditions and might optimize them out, but I haven't double-checked that. We might not want to rely on that optimization, though.

Reply:
IIRC removal of the saveLayer would cause the pixel snapping no?

Reply:
(locally I notice that there is no pixel snapping with this change)

Reply:
I don't think saveLayer does pixel snapping. The pixel snapping happens when we execute the call to `GetIntegralTransCTM` inside the `SUPPORT_FRACTIONAL_TRANSLATION` ifdefs (the way we use that method is its own nest of issues, but that is where the snapping happens).